### PR TITLE
Git hub ssh instructions

### PR DESCRIPTION
**Pull request by edenspring**
Adjusted wording in guide to better reflect a/A standards. Added section on creating config file for SSH to provide better support for users that rename their identity file or make use of a passphrase. Added clarification information  to github-setup.md to better explain what SSH is.

### CODE REVIEW

File: github-setup.md [hunk 1]
```diff
@@ -77,4 +77,4 @@ Once our secret manager is setup, we want to **restart our terminal/WSL Shell**.
...
 
 ### Still having trouble?
-We can make use of another form of authentication, however you may run into some unanticipated issues depending on your network setup. If you'd like to explore this option, please checkout [this guide](https://hackmd.io/cZQaolzKS0iEK9bkJYSWgA)
+We can make use of another form of authentication, however you may run into some unanticipated issues depending on your network setup. GitHub supports SSH authentication. SSH stands for Secure SHell and is an older asynchronous cryptography system of authentication that generates private identifier data and public identifier data. The private data stays on your local machine and is validated against the public part of the record to verify your identity. This provides a way of securely operating network services over an unsecured network(such as the internet at large). If you'd like to explore this option, please checkout [this guide](https://hackmd.io/@AgDXdHgSSPKsJIhCxlaTuA/BJtNu88fF)
```
Comment (CalebBraaten) [inline github-setup.md]: You can keep this short and sweet. "If you have trouble with this setup or a preference for ssh you can follow [this guide]."

**Comment by CalebBraaten** [comment]
You're linking to an external resource. Add the SSH guide as a file to this repository.

**Comment by CalebBraaten** [comment]
The resource you link too, should have its language updated to have less narrative and be a more step by step guide.

**Comment by edenspring** [comment]
Sounds good, I will get those changes pushed soon!

**Pull request merged**